### ISSUE TITLE
Move to python-docs-samples

### ISSUE THREAD

**Issue by theacodes** [GoogleCloudPlatform/storage-signedurls-python#9]
Is there any reason why this shouldn't be consolidated into GoogleCloudPlatform/python-docs-samples?

**Comment by jterrace** [comment]
Sure, I don't see why not.

**Comment by theacodes** [comment]
Do you happen to know where this sample is used in the docs?

**Comment by jterrace** [comment]
It's not used in the docs.

**Comment by theacodes** [comment]
Hmm, what is it used for?

**Comment by jterrace** [comment]
It's just an example for customers to see how to generate signed URLs.

**Comment by theacodes** [comment]
Gotcha. So, gcloud-python can now [do this for you](http://googlecloudplatform.github.io/gcloud-python/stable/storage-blobs.html#gcloud.storage.blob.Blob.generate_signed_url). Do we still need this repo or can we move it to googlearchive?

**Comment by jterrace** [comment]
The purpose of the demo is to show how to generate the URLs programmatically, not as a tool for generating them on the command line. I don't think the code in gcloud is easily extractable and re-usable.

**Comment by theacodes** [comment]
`gcloud-python` is the client library for Google Cloud APIs, not the command line tool (though I completely understand the confusion).

**Comment by jterrace** [comment]
Ah, I see. In that case, I think we can archive this repo. However, we should make sure a generate_signed_url example is in our docs.

**Comment by theacodes** [comment]
Fair enough. I'll make a doc issue for that and I'll deprecate this once that's done.
